plugins/image-prioritizer/class-image-prioritizer-video-tag-visitor.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Tag visitor that optimizes VIDEO tags:
+ * - Adds preload links for poster images if in a breakpoint group's LCP.
+ *
+ * @package image-prioritizer
+ *
+ * @since n.e.x.t
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Image Prioritizer: Image_Prioritizer_Video_Tag_Visitor class
+ *
+ * @since n.e.x.t
+ *
+ * @access private
+ */
+final class Image_Prioritizer_Video_Tag_Visitor extends Image_Prioritizer_Tag_Visitor {
+
+	/**
+	 * Visits a tag.
+	 *
+	 * @param OD_Tag_Visitor_Context $context Tag visitor context.
+	 *
+	 * @return bool Whether the tag should be tracked in URL metrics.
+	 */
+	public function __invoke( OD_Tag_Visitor_Context $context ): bool {
```

Comment:
@westonruter I suppose this is where I would add the logic for #1575?

Reply:
Yes exactly!